content/bn/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,136 @@
+---
+title: গার্বেজ কালেকশন
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+{{<glossary_definition term_id="garbage-collection" length="short">}}
+গার্বেজ কালেকশন সিস্টেমকে নিম্নলিখিত রিসোর্সগুলি পরিষ্কার করতে দেয়:
+
+* [সমাপ্ত পডগুলি](/bn/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+* [সম্পন্ন হওয়া জবগুলি](/bn/docs/concepts/workloads/controllers/ttlafterfinished/)
+* [মালিক রেফারেন্স ছাড়া অবজেক্ট](#owners-dependents)
+* [অব্যবহৃত কন্টেইনার এবং কন্টেইনার ইমেজগুলি](#containers-images)
+* [ডায়নামিকভাবে প্রস্তুতকৃত, StorageClass এর জন্য পুনরুদ্ধার নীতির সাথে Delete করা PV](/bn/docs/concepts/storage/persistent-volumes/#delete)
```

Comment:
Use the English (`en`) version as a reference  `/docs/concepts/storage/persistent-volumes/#delete` .  

Once the Bengali version is created, you can replace the English link with:
`/bn/docs/concepts/storage/persistent-volumes/#delete`.